app/util/GitHubUtil.scala

```diff
@@ -0,0 +1,24 @@
+package util
+
+import java.io.FileNotFoundException
+
+import scala.io.Source
+
+object GitHubUtil {
+
+  private val identifier = "A-Za-z0-9-_"
+  private val gitHubUrlPattern = s"""http(s)?://github.com/[$identifier]+/[$identifier]+(/)?""".r.pattern
+  private val readmeUrl = "https://raw.githubusercontent.com/%s/%s/master/README.md"
```

Comment:
Have you tested this with varying capitalisation?
- `readme.md`
- `README.MD`
- `readme.MD`

Additionally, GitHub supports more than just `.md` as a file extension for markdown: https://github.com/github/markup/blob/master/lib/github/markup/markdown.rb#L32

Reply:
This currently only supports `README.md`. I'll update it with different types soon.